project/profiles/capacity4more/modules/c4m/content/c4m_content_document/c4m_content_document.features.field_base.inc

```diff
@@ -95,5 +72,40 @@ function c4m_content_document_field_default_field_bases() {
     'type' => 'entityreference',
   );
 
+  // Exported field_base: 'field_c4m_document'.
+  $field_bases['field_c4m_document'] = array(
```

Comment:
@guytepper 
This one seems wrong, should be `c4m_document` without the `field_` prefix